mautic/mautic
# Fix report empty date value

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
This cause to us display same date values If `$cellVal` was empty



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. I don't know how to reproduce. I noticed it on one instance, and make quick fix. Code review should be good.




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10752"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
